evtx.go

```diff
@@ -385,6 +401,22 @@ func (self *ParseContext) ConsumeUint64() uint64 {
 	return result
 }
 
+func (self *ParseContext) ConsumeReal64() (ret float64) {
+
+	if len(self.buff) < self.offset+8 {
+		return 0
+	}
+
+	buf := bytes.NewReader(self.buff[self.offset:])
+	err := binary.Read(buf, binary.LittleEndian, &ret)
+	if err != nil {
+		fmt.Println("Binary read for Real64 Failed:", err)
+	}
```

Comment:
```suggestion
	if err != nil {
		return 0
	}
```

you should prefer feedback from the project owners - I'm just an innocent bystander :-)

Reply:
Thank you for reviewing! I concur with your suggestions ☺️